common.mk

```diff
@@ -146,19 +146,12 @@ clean-versions:
 
 generate-all: generate
 
-MANIFEST_FILE ?= manifest.sh
-
-auto_targets.mk: $(MANIFEST_FILE)
-	MANIFEST_FILE="$(MANIFEST_FILE)" \
-	VERSIONS="$(VERSIONS)" \
-	SKIP_GENERATOR_FOR="${SKIP_GENERATOR_FOR}" \
-	$(generator)
-
-# triggers build of auto_targets.mk automatically
--include auto_targets.mk
-
-# We have to remove auto_targets.mk here, otherwise subsequent make calls
-# with different VERSIONS=* option keeps the auto_targets.mk unchanged.
-.PHONY: generate
-generate: $(DISTGEN_TARGETS) $(DISTGEN_MULTI_TARGETS) $(COPY_TARGETS) $(SYMLINK_TARGETS)
-	rm auto_targets.mk
+.PHOHY: generate
+generate:
+	for version in ${VERSIONS} ; do \
+		if [[ "$$version" == *-minimal ]]; then \
+			$(generator) -v $$version -m manifest-minimal.yml -s specs/multispec.yml ; \
```

Comment:
would it be possible to add there directly `python3` only for sure if python2 is installed e.g. on CentOS 7 and we did not install python3

Reply:
The first line of the script is "#!/usr/bin/env python3", there is no way to execute it with Python 2.